tests/test_helpers.py

```diff
@@ -22,6 +22,9 @@
 from werkzeug.http import parse_cache_control_header, parse_options_header
 from werkzeug.http import http_date
 from flask._compat import StringIO, text_type
+from hypothesis import given
```

Comment:
Use `import hypothesis`, `@hypothesis.given`, otherwise this is going to get even harder to keep track of.  Same with below: `import hypothesis.strategies`, `hypothesis.strategies.uuids()`, etc.